src/test/rustdoc/reexport-check.rs

```diff
@@ -6,6 +6,7 @@ extern crate reexport_check;
 // @!has 'foo/index.html' '//code' 'pub use self::i32;'
 // @has 'foo/index.html' '//tr[@class="module-item"]' 'i32'
 // @has 'foo/i32/index.html'
+#[allow(deprecated, deprecated_in_future)]
```

Comment:
I think this is the right thing to do - `i32` overlaps with the primitive namespace and this test is checking that the it still shows up as a module.

cc @GuillaumeGomez

Reply:
I'm fine with it too.